piety-promoted-v3/modernized/10-chapter-10.adoc

```diff
@@ -175,6 +175,10 @@ then having lately acquired the crown of Hungary,
 to which letter her companions added also their names.
 It is as follows:
 
+[.embedded-content-document.letter]
+--
+
+[.salutation]
```

Comment:
I debated if this should be a salutation or a letter heading. 🤔

Reply:
I think it would have been OK either way, but I slightly lean towards making it a salutation, like you did. 👍